app/controllers/tabs/tabs_controller.rb

```diff
@@ -78,6 +78,13 @@ def update
     end
   end
 
+  def sort
+    params[:order].each do |key, value|
```

Comment:
Unused block argument - `key`. If it's necessary, use `_` or `_key` as an argument name to indicate that it won't be used.